webapp-django/crashstats/settings/base.py

```diff
@@ -30,7 +30,7 @@
     'django.contrib.sessions',
     'django.contrib.staticfiles',
     'commonware.response.cookies',
-    'south',  # important that django_nose comes AFTER this one
+    # 'south',  # important that django_nose comes AFTER this one
```

Comment:
A perfectly good review comment. 
I started that way but clearly didn't get back to removing it properly. 
Thanks!